### DISCUSSION TITLE
Step Into (F8 or F11)

### DISCUSSION THREAD

**Discussion by mayost** [twinbasic/twinbasic#1808] [Q&A]
Hello

When I enter the code:

Private Sub Form_Load()
    Dim lngVar as Long
    do until lngVar=10
        Debug.print lngVar
    loop
    Stop
    End
end Sub

When it reaches: STOP
It does stop
But if I continue and press F8 to move forward one step, it reaches the END command and if I press F8 again to move forward, the ID gets stuck and does not allow anything to continue, it is on pause

Thanks

**Comment by fafalone** [comment]
Transferred to issues.

https://github.com/twinbasic/twinbasic/issues/1809
